lm_eval/tasks/klue.py

```diff
@@ -33,27 +33,29 @@ def has_test_docs(self):
 
     def training_docs(self):
         if self._training_docs is None:
-            self._training_docs = list(self.dataset["train"])
+            self._training_docs = list(map(self._process_doc,self.dataset["train"]))
         return self._training_docs
 
     def validation_docs(self):
-        return self.dataset["validation"]
+        return map(self._process_doc,self.dataset["validation"])
 
     def _process_doc(self, doc):
         out_doc = {
+            "title": doc["title"],
             "choices": ["과학", "경제", "사회", "생활", "세계", "스포츠", "정치"],
+			"gold": doc["label"]
```

Comment:
Tiny nit: Could you unindent this to align it with the rest of the `out_doc` fields?